### ISSUE TITLE
cryoatom did not produce the expected results

### ISSUE THREAD

**Issue by Hai-W-L** [YangLab-SDU/CryoAtom#2]
I am using cryoatom for automated modeling of my map and encountered an issue. I would appreciate some guidance.

After running the program, the output files were problematic: `ex_raw.cif` (1086 KB) and `ex.cif` (1 KB). As shown in the attached image, they appear to be discontinuous peptide fragments.

My map is reported at 3.55 Å resolution in cryoSPARC, and visual inspection confirms that the map quality is not sufficient to clearly distinguish features like helices.

I also ran the official example (`cryoatom build -s rcsb_pdb_7XMV.fasta -v emd_33306.map -o out`), which successfully produced a normal structure in `out.cif` (4777 KB).

Therefore, my questions are:
1.  Is the primary issue likely the low quality/resolution of my map?
2.  Are there any other parameters or steps I could adjust to potentially improve the results for this map? (I tried adding a mask, but the outcome was the same.)

For your information, my `pyhmmer` version is 0.7.1.

Thank you for your time and help.

<img width="1019" height="432" alt="Image" src="https://github.com/user-attachments/assets/75eb6d72-3fed-48f4-a414-4634fc474876" />

(CryoAtom) [spuser@spgpu RAPiDock]$ cryoatom build -s "****/ex.fasta" -v "/cryosparc_P25_J483_008_volume_map_sharp.mrc" -o /1215
---------------------------- CryoAtom -----------------------------
By Baoquan Su, Yang lab.
--------------------- CryoAtom Stage1 (Predict C-alpha atoms by U-Net) ---------------------
  0%|                                                                                                                                                               | 0/125.0 [00:00<?, ?it/s]/home/spuser/.conda/envs/CryoAtom/lib/python3.8/site-packages/torch/nn/modules/conv.py:605: UserWarning: Applied workaround for CuDNN issue, install nvrtc.so (Triggered internally at ../aten/src/ATen/native/cudnn/Conv_v8.cpp:80.)
  return F.conv3d(
/home/spuser/.conda/envs/CryoAtom/lib/python3.8/site-packages/torch/utils/checkpoint.py:429: UserWarning: torch.utils.checkpoint: please pass in use_reentrant=True or use_reentrant=False explicitly. The default value of use_reentrant will be updated to be False in the future. To maintain current behavior, pass use_reentrant=True. It is recommended that you use use_reentrant=False. Refer to docs for more details on the differences between the two variants.
  warnings.warn(
220it [00:24,  9.16it/s]
------------------ CryoAtom Stage2 (Build Protein Model by Cry-Net), round 1 / 3 ------------------
  0%|                                                                                                                                                                | 0/7023 [00:00<?, ?it/s]/home/spuser/.conda/envs/CryoAtom/lib/python3.8/site-packages/torch/utils/checkpoint.py:429: UserWarning: torch.utils.checkpoint: please pass in use_reentrant=True or use_reentrant=False explicitly. The default value of use_reentrant will be updated to be False in the future. To maintain current behavior, pass use_reentrant=True. It is recommended that you use use_reentrant=False. Refer to docs for more details on the differences between the two variants.
  warnings.warn(
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 7023/7023 [02:29<00:00, 47.03it/s]
------------------ CryoAtom Stage2 (Build Protein Model by Cry-Net), round 2 / 3 ------------------
  0%|                                                                                                                                                                | 0/6525 [00:00<?, ?it/s]/home/spuser/.conda/envs/CryoAtom/lib/python3.8/site-packages/torch/utils/checkpoint.py:429: UserWarning: torch.utils.checkpoint: please pass in use_reentrant=True or use_reentrant=False explicitly. The default value of use_reentrant will be updated to be False in the future. To maintain current behavior, pass use_reentrant=True. It is recommended that you use use_reentrant=False. Refer to docs for more details on the differences between the two variants.
  warnings.warn(
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 6525/6525 [02:22<00:00, 45.91it/s]
------------------ CryoAtom Stage2 (Build Protein Model by Cry-Net), round 3 / 3 ------------------
  0%|                                                                                                                                                                | 0/5991 [00:00<?, ?it/s]/home/spuser/.conda/envs/CryoAtom/lib/python3.8/site-packages/torch/utils/checkpoint.py:429: UserWarning: torch.utils.checkpoint: please pass in use_reentrant=True or use_reentrant=False explicitly. The default value of use_reentrant will be updated to be False in the future. To maintain current behavior, pass use_reentrant=True. It is recommended that you use use_reentrant=False. Refer to docs for more details on the differences between the two variants.
  warnings.warn(
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5991/5991 [02:04<00:00, 48.14it/s]
----------------------------------------------------------------------
CryoAtom has completed the construction of the full-atom protein model.
----------------------------------------------------------------------
You can find the final model filtered by the fasta sequence at: 1215/1215.cif.

The confidence scores of the model predictions are saved in the bfactor field of the mmcif file.
----------------------------------------------------------------------
If you want to see more of the built regions in the map, you can go to: 1215/1215_raw.cif
However, it often has many untreated unknown regions
(which may be caused by the noise in the density map itself).
----------------------------------------------------------------------
done!
